bin/improver-tests

```diff
@@ -18,46 +18,94 @@ function echo_ok {
     echo -e "\033[1;32m[OK]\033[0m $1"
 }
 
-if [[ ${1:-} == '--help' ]] || [[ ${1:-} == '-h' ]]; then
+function improver_test_pep8 {
+    # PEP8 testing.
+    ${PEP8:-pep8} improver
+    echo_ok "pep8"
+}
+
+function improver_test_pylint {
+    # Pylint obvious-errors-only testing.
+    ${PYLINT:-pylint} --rcfile=../etc/pylintrc improver
+}
```

Comment:
Ah, yes, forgot about that!  (Perhaps in the future we could create a wrapper script which checks that the pylint score is >= some threshold (8?) and fails if not?)